web/front-end/src/config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "SERVER_URL": "https://zsi1on0bi0.execute-api.us-west-2.amazonaws.com/api",
```

Comment:
Is this something the user is going to change?

Reply:
Yes, they will replace this URL with the current output from `chalice url`
I can replace this url with some meta-tag like  `<replace-with-chalice-url>` if you wish